package.json

```diff
@@ -30,6 +35,12 @@
     "name": "Jérémie Astori",
     "url": "https://jeremie.astori.fr"
   },
+  "contributors": [
+    {
+      "name": "Ivan Zlatev",
+      "url": "http://ivanz.com"
+    }
+  ],
```

Comment:
I'm not 100% sure about this, but I do appreciate the help from contributors a lot!
For those who are wondering, I'll add other contributor names once this PR is merged, and if I happen to ever receive hundreds of contributions, I'll rethink this 😄.